ethereum/contracts/query/QueryResponse.sol

```diff
@@ -86,6 +86,31 @@ struct SolanaAccountResult {
     bytes data;
 }
 
+// @dev SolanaPdaQueryResponse describes a Solana PDA query per-chain query.
+struct SolanaPdaQueryResponse {
+    bytes requestCommitment;
```

Comment:
I think the documentation could be improved here, especially since this is solana-specific language in an EVM program :)